src/views/Timeline.vue

```diff
@@ -121,15 +135,32 @@ export default {
             }
           }
         );
-        this.proposals = proposals.timeline;
+        this.stopLoadingMore = proposals.timeline.length < loadBy;
+        this.proposals = this.proposals.concat(proposals.timeline);
       } catch (e) {
         console.log(e);
       }
-      this.loading = false;
+    },
+    async loadMoreProposals() {
+      if (!this.stopLoadingMore && this.proposals[0]) {
+        this.loadingMore = true;
+        await this.loadProposals(this.limit);
+        this.limit += loadBy;
+        this.loadingMore = false;
+      }
     }
   },
   async created() {
+    this.loading = true;
     await this.loadProposals();
+    this.loading = false;
+  },
+  mounted() {
+    const el = document.getElementById('endsensor');
```

Comment:
We should avoid having duplicate code, would be better if we can move the logic into a helper function and reuse that function instead.

Reply:
I thought about it, but couldn't come up with a way that was feasible.

Reply:
How would I call the `this.loadMoreProposals();` function from inside the helper?

Reply:
Ok, I think I found a solution with mixins or inherit.